src/shared/components/enhancedReactTable/IEnhancedReactTableProps.tsx

```diff
@@ -0,0 +1,24 @@
+import {IColumnFormatter, IColumnSortFunction, IColumnFilterFunction} from "./IColumnFormatterProps";
```

Comment:
this file doesn't have any jsx in it so it should be a .ts file, just so compiler doesn't have waste time looking for jsx